wandb/sdk/launch/docker.py

```diff
@@ -97,6 +96,7 @@ def build_docker_image(project: _project_spec.Project, repository_uri, base_imag
         api_key=api.api_key,
         wandb_project=wandb_project,
         wandb_entity=wandb_entity,
+        uid=os.getuid(),
```

Comment:
We may want to make this configurable.  I believe the jupyter base images always create a user named jovyan with UID 1000.  They then add that user to the root group, we should likely just piggy back on that and let the user change the uid if they choose to.

Reply:
Wasn't entirely sure what the right way to piggy back here is. So I made the UID configurable but default to using the users uid. Would be happy to clarify if you have something else in mind.